docs/checks/internal/checks.yaml

```diff
@@ -746,47 +747,29 @@ checks:
       Risk: `Low` (possible impediment to security review)
 
       This check tries to determine if the project has published a license. It
-      works by using either hosting APIs or by checking standard locations
-      for a file named according to common conventions for licenses.
+      works by checking standard locations for a file named according to common
```

Comment:
You've reverted some changes to the License check in multiple places in this file. Make sure you've rebased, merged those changes into this branch.

Reply:
thought I had rebased. But apparently not! Thanks